pkg/internal/appolly/appolly.go

```diff
@@ -124,17 +123,13 @@ func setupKubernetes(ctx context.Context, ctxInfo *global.ContextInfo) {
 		return
 	}
 
-	informer, err := ctxInfo.K8sInformer.Get(ctx)
+	// caching the database initialization. Discarding it as subsequent
+	// invocations to Get will return the same initialized instance
+	_, err := ctxInfo.K8sInformer.Get(ctx)
```

Comment:
Completely subjective and arguable suggestion: perhaps we can do:

```
func refreshK8sInformerCache(ctx ...) error {
   _, err: ctxInfo.K8sInformer.Get(ctx) // force the cache to be populated/refreshed
  return err
} 

err := refreshK8sInformerCache(ctx)
```

just so it is a tad more idiomatic - but like I said, feel free to ignore as this is a very subjective approach